apps/jobs/views.py

```diff
@@ -335,14 +340,19 @@ def leaderboard(request, challenge_phase_split_id):
     # while populating leaderboard
     challenge_obj = challenge_phase_split.challenge_phase.challenge
     challenge_hosts_emails = challenge_obj.creator.get_all_challenge_host_email()
+    # challenge_host = challenge_obj.creator.team_name
     is_challenge_phase_public = challenge_phase_split.challenge_phase.is_public
     # Exclude the submissions from challenge host team to be displayed on the leaderboard of public phases
     challenge_hosts_emails = [] if not is_challenge_phase_public else challenge_hosts_emails
 
     challenge_host_user = is_user_a_host_of_challenge(request.user, challenge_obj.pk)
 
     leaderboard_data = LeaderboardData.objects.exclude(
-        submission__created_by__email__in=challenge_hosts_emails)
+        Q(submission__created_by__email__in=challenge_hosts_emails) & Q(submission__baseline_submission=False))
```

Comment:
I don't think we need to modify it. Can you please state why we need this?

Reply:
This is to enable Submission by `Host`, if and only if the submission is `baseline submission`.

Reply:
I didn't get this, we only want the submission to appear on the leaderboard when the baseline submission flag is True. That's it.